From ba5fde54099bcc2dab67c82ecae8a4d4d03e557b Mon Sep 17 00:00:00 2001
From: Ariel D'Alessandro <ariel.dalessandro@collabora.com>
Date: Fri, 10 Jan 2025 14:57:41 -0300
Subject: [PATCH] usb: gadget: ether: Set USB connect timeout to 20s

Genio 350 EVK board often timeouts on a tftp boot sequence reporting:

```
The remote end did not respond in time.
```

Set the default timeout value to 20s, which is enough for these cases.

Signed-off-by: Ariel D'Alessandro <ariel.dalessandro@collabora.com>
---
 drivers/usb/gadget/ether.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/usb/gadget/ether.c b/drivers/usb/gadget/ether.c
index 11b1a6221b..3e40d66467 100644
--- a/drivers/usb/gadget/ether.c
+++ b/drivers/usb/gadget/ether.c
@@ -93,7 +93,7 @@ static const char driver_desc[] = DRIVER_DESC;
 			|USB_CDC_PACKET_TYPE_PROMISCUOUS \
 			|USB_CDC_PACKET_TYPE_DIRECTED)
 
-#define USB_CONNECT_TIMEOUT (3 * CONFIG_SYS_HZ)
+#define USB_CONNECT_TIMEOUT (20 * CONFIG_SYS_HZ)
 
 /*-------------------------------------------------------------------------*/
 
